libpdbg/device.c

```diff
@@ -701,6 +701,73 @@ static void pdbg_targets_init_virtual(struct pdbg_target *node, struct pdbg_targ
 	}
 }
 
+/**
```

Comment:
fix the commit message it is > 55 characters or so

Reply:
Corrected